js/search.js

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/*globals ContentSearchUIController*/
+"use strict";
```

Comment:
I don't know if we need this file.... we should see if we can export `ContentSearchUIController` without needing a constructor.